scripting/pugsetup/natives.sp

```diff
@@ -501,7 +512,18 @@ public int Native_SetTeamBalancer(Handle plugin, int numParams) {
   bool override = GetNativeCell(2);
   if (!PugSetup_IsTeamBalancerAvaliable() || override) {
     g_BalancerFunctionPlugin = plugin;
-    g_BalancerFunction = view_as<TeamBalancerFunction>(GetNativeFunction(1));
+    ArrayList players = new ArrayList();
```

Comment:
Why is this changing? If a function handle can't be stored like this anymore, these natives probably will need to be deleted and replaced with a forward to prevent misuse.

For example, the pugsetup_rwsbalancer plugin calls this native during OnAllPluginsLoaded(): https://github.com/splewis/csgo-pug-setup/blob/3f217bda97e9fffd745f6248bf15e4c44dbec36e/scripting/pugsetup_rwsbalancer.sp#L100

No players will be ready at that point, so this effectively breaks that plugin.